p2p/stream/protocols/sync/const.go

```diff
@@ -0,0 +1,17 @@
+package sync
+
+import "time"
+
+const (
+	// GetBlockHashesAmountCap is the cap of GetBlockHashes reqeust
+	GetBlockHashesAmountCap = 50
+
+	// GetBlocksByNumAmountCap is the cap of request of a single GetBlocksByNum request
+	GetBlocksByNumAmountCap = 10
```

Comment:
can you add some additional comments here why 10?  I remember there was a max size limit.

Reply:
Right, it is better to add some comments here. BTW, max size limit is actually calculated based on this number, which is  just an arbitrary number I believe reasonable.